diff -up ./build.xml.gcj-javadoc ./build.xml
--- ./build.xml.gcj-javadoc	2010-07-23 21:41:42.000000000 +0200
+++ ./build.xml	2010-07-23 21:43:12.161570259 +0200
@@ -570,8 +570,6 @@
       <group title="Java Native Access" packages="com.sun.jna,com.sun.jna.ptr,com.sun.jna.types,com.sun.jna.win32"/>
       <group title="Platform Utilities" packages="com.sun.jna.platform,com.sun.jna.platform.dnd"/>
       <group title="Platform Specific" packages="com.sun.jna.platform.*"/>
-    	
-      <arg value="-notimestamp"/>
     </javadoc>
   </target>
 
diff -up ./src/com/sun/jna/Function.java.gcj-javadoc ./src/com/sun/jna/Function.java
--- ./src/com/sun/jna/Function.java.gcj-javadoc	2009-10-24 17:43:53.000000000 +0200
+++ ./src/com/sun/jna/Function.java	2010-07-23 21:41:42.148570758 +0200
@@ -76,7 +76,7 @@ public class Function extends Pointer {
      *                  Library in which to find the native function
      * @param   functionName
      *                  Name of the native function to be linked with
-     * @throws {@link UnsatisfiedLinkError} if the library is not found or
+     * @throws UnsatisfiedLinkError if the library is not found or
      * the given function name is not found within the library.
      */
     public static Function getFunction(String libraryName, String functionName) {
@@ -97,7 +97,7 @@ public class Function extends Pointer {
      * @param   callFlags
      *                  Function <a href="#callflags">call flags</a>
      *                  
-     * @throws {@link UnsatisfiedLinkError} if the library is not found or
+     * @throws UnsatisfiedLinkError if the library is not found or
      * the given function name is not found within the library.
      */
     public static Function getFunction(String libraryName, String functionName, int callFlags) {
@@ -161,7 +161,7 @@ public class Function extends Pointer {
      *                 Name of the native function to be linked with
      * @param  callFlags
      *                 Function <a href="#callflags">call flags</a>
-     * @throws {@link UnsatisfiedLinkError} if the given function name is
+     * @throws UnsatisfiedLinkError if the given function name is
      * not found within the library.
      */
     Function(NativeLibrary library, String functionName, int callFlags) {
